cou/apps/app.py

```diff
@@ -38,7 +39,7 @@
 class AppFactory:
     """Factory class for Application objects."""
 
-    apps_type: Dict[str, Type[Application]] = {}
+    apps_type: dict[str, type[Application]] = {}
```

Comment:
Why we are changing it from `typing.Dict` to `dict` and `typing.Type` to `type`?

Reply:
I've been reading about `typing` and on [PEP 585](https://peps.python.org/pep-0585/) Dict, List, Type, Set and etc will be [deprecated](https://docs.python.org/3/library/typing.html#deprecated-aliases) from python 3.14. I think we should use the built-in types from now on.

Reply:
Thanks, that make sense.